src/Data/Argonaut/Decode/Generic/Rep.purs

```diff
@@ -95,7 +125,7 @@ mFail msg = maybe (Left msg) Right
 
 -- | A function for decoding `Generic` sum types using string literal representations
 decodeLiteralSum :: forall a r. Rep.Generic a r => DecodeLiteral r => Json -> Either String a
-decodeLiteralSum = decodeLiteralSumWithTransform id
+decodeLiteralSum = decodeLiteralSumWithTransform (\a -> a)
```

Comment:
`identity`?

Reply:
ah yes ... it took me some time to find it and my tooling was broken ... I'll fix it